tasks/synopsys-detect-task/detect-task.ts

```diff
@@ -0,0 +1,116 @@
+import * as task from 'azure-pipelines-task-lib/task'
+import * as os from 'os'
+import {IBlackduckConfiguration} from "./ts/model/IBlackduckConfiguration"
+import {IDetectConfiguration} from "./ts/model/IDetectConfiguration"
+import {ITaskConfiguration} from "./ts/model/ITaskConfiguration"
+import {DetectADOConstants} from "./ts/DetectADOConstants"
+import {IProxyInfo} from "./ts/model/IProxyInfo"
+import {DetectScript} from "./ts/DetectScript";
+import {PowershellDetectScript} from "./ts/PowershellDetectScript";
+import {BashDetectScript} from "./ts/BashDetectScript";
+const winston = require("winston")
+import fileSystem from "fs";
+
+const log = winston.createLogger({
```

Comment:
Should I use some logger or just stick to console? Console.log prints to the azure devops console

Reply:
Do they give the same output to the user? If so I'd be inclined to use a logger so that we could potentially expose a way to change the log-level.

Reply:
I agree with gavin, it gives us more control. We can also add a timestamp or further customize it later.